tutorials/00-introduction/00_introduction.jmd

```diff
@@ -106,31 +114,51 @@ animation = @gif for (i, N) in enumerate(Ns)
+
+For the mathematically inclined, the $\operatorname{Beta}$ distribution is updated by adding each coin flip to the parameters $\alpha$ and $\beta$ of the distribution.
+Initially, the parameters are defined as $\alpha = 1$ and $\beta = 1$.
+Over time, with more and more coin flips, $\alpha$ and $\beta$ will be approximately equal to each other as we are equally likely to flip a heads or a tails.
+
+The mean of the $\operatorname{Beta}(\alpha, \beta)$ distribution is
+
+$$\operatorname{E}[X] = \dfrac{\alpha}{\alpha+\beta}.$$
+
+This implies that the plot of the distribution will become centered around 0.5 for a large enough number of coin flips, as we expect $\alpha \approx \beta$.
+
+The variance of the $\operatorname{Beta}(\alpha, \beta)$ distribution is
+
+$$\operatorname{var}[X] = \dfrac{\alpha\beta}{(\alpha + \beta)^2 (\alpha + \beta + 1)}.$$
+
+Thus the variance of the distribution will approach 0 with more and more samples, as the denominator will grow faster than will the numerator.
+More samples means less variance.
+This implies that the distribution will reflect less uncertainty about the probability of receiving a heads and the plot will become more tightly centered around 0.5 for a large enough number of coin flips.
 
 ### Coin Flipping With Turing
 
-In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior). Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
+In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior).
+Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
 
-We are now going to move away from the closed-form expression above and specify the same model using **Turing**. To do so, we will first need to import `Turing`, `MCMCChains`, `Distributions`, and `StatPlots`. `MCMCChains` is a library built by the Turing team to help summarize Markov Chain Monte Carlo (MCMC) simulations, as well as a variety of utility functions for diagnostics and visualizations.
+We are now going to move away from the closed-form expression above and specify the same model using **Turing**.
+To do so, we will first need to load `Turing`.
 
 ```julia
-# Load Turing and MCMCChains.
-using Turing, MCMCChains
+using Turing
+```
 
-# Load the distributions library.
-using Distributions
+Additionally, we load `MCMCChains`.
+It is a library built by the Turing team to help summarize Markov Chain Monte Carlo (MCMC) simulations, as well as a variety of utility functions for diagnostics and visualizations.
```

Comment:
```suggestion
```

Probably also too technical for one of the first tutorials.

Reply:
I reformulated it.